<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*-----QWidget-----*/
QWidget
{
	background-color: #232430;
	color: #000000;
	border-color: #000000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: #232430;
	color: #c1c1c1;
	border-color: #000000;

}


/*-----QPushButton-----*/
QPushButton
{
	background-color: #ff9c2b;
	color: #000000;
	font-weight: bold;
	border-style: solid;
	border-color: #000000;
	padding: 6px;

}


QPushButton::hover
{
	background-color: #ffaf5d;

}


QPushButton::pressed
{
	background-color: #dd872f;

}


/*-----QToolButton-----*/
QToolButton
{
	background-color: #ff9c2b;
	color: #000000;
	font-weight: bold;
	border-style: solid;
	border-color: #000000;
	padding: 6px;

}


QToolButton::hover
{
	background-color: #ffaf5d;

}


QToolButton::pressed
{
	background-color: #dd872f;

}


/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #38394e;
	color: #c1c1c1;
	border-style: solid;
	border-width: 1px;
	border-color: #4a4c68;

}


/*-----QTableView-----*/
QTableView, 
QHeaderView, 
QTableView::item 
{
	background-color: #232430;
	color: #c1c1c1;
	border: none;

}


QTableView::item:selected 
{ 
    background-color: #41424e;
    color: #c1c1c1;

}


QHeaderView::section:horizontal 
{
    background-color: #232430;
	border: 1px solid #37384d;
	padding: 5px;

}


QTableView::indicator{
	background-color: #1d1d28;
	border: 1px solid #37384d;

}


QTableView::indicator:checked{
	image:url(&quot;./ressources/check.png&quot;); /*To replace*/
	background-color: #1d1d28;

}

/*-----QTabWidget-----*/
QTabWidget::pane 
{ 
    border: none;

}


QTabWidget::tab-bar 
{
    left: 5px; 

}


QTabBar::tab 
{
    color: #c1c1c1;
    min-width: 1px;
	padding-left: 25px;
	margin-left:-22px;
    height: 28px;
	border: none;

}


QTabBar::tab:selected 
{
    color: #c1c1c1;
	font-weight: bold;
    height: 28px;

}


QTabBar::tab:!first 
{
    margin-left: -20px;

}


QTabBar::tab:hover 
{
    color: #DDD;

}


/*-----QScrollBar-----*/
QScrollBar:horizontal 
{
    background-color: transparent;
    height: 8px;
    margin: 0px;
    padding: 0px;

}


QScrollBar::handle:horizontal 
{
    border: none;
	min-width: 100px;
    background-color: #56576c;

}


QScrollBar::add-line:horizontal, 
QScrollBar::sub-line:horizontal,
QScrollBar::add-page:horizontal, 
QScrollBar::sub-page:horizontal 
{
    width: 0px;
    background-color: transparent;

}


QScrollBar:vertical 
{
    background-color: transparent;
    width: 8px;
    margin: 0;

}


QScrollBar::handle:vertical 
{
    border: none;
	min-height: 100px;
    background-color: #56576c;

}


QScrollBar::add-line:vertical, 
QScrollBar::sub-line:vertical,
QScrollBar::add-page:vertical, 
QScrollBar::sub-page:vertical 
{
    height: 0px;
    background-color: transparent;

}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>207</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="hourcomboBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="mincomboBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>27</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>31</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>33</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>34</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>35</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>36</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>37</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>39</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>40</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>41</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>42</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>43</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>44</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>45</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>46</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>47</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>48</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>49</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>51</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>52</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>53</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>54</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>55</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>56</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>58</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>59</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="setButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#setButton:hover {
	
	background-color: rgb(255, 255, 255);
 }</string>
   </property>
   <property name="text">
    <string>SET</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Current Time</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid;
color: rgb(255, 255, 255);
border-color: rgb(255, 156, 43);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#removeButton:hover {
		
	background-color: rgb(255, 255, 255);
 }</string>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
